### ISSUE TITLE
Refactor hardcoded string literals to i18n format

### ISSUE THREAD

**Issue by prathamVaidya** [WikiEducationFoundation/WikiEduDashboard#5738]
### Current Behavior:

The WikiEduDashboard codebase uses `i18n` to support localization. Localization helps translate the website text into multiple languages. Though the codebase has very high localization support, there are more than 300+ warnings related to hardcoded string literals.

### Expected Behavior:

These hardcoded string literals should be moved to i18n format to enable localization. This is a good first issue for beginners. Once you have [setup the development environment](https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/docs/setup.md), run the following command :

```
yarn coverage
```

The command will take 1-2 mins to complete and will list all the components that require refactoring. Then you can pick up one component and try to move the hardcoded string to `i18n`.

## Example of how to do it

Before:

![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/93db73f6-9aa5-4980-bd1c-c27f340d1e52)

After:

![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/ac4452e3-722d-4ead-b655-3935b3bb1680)

Extract the text and add it in `en.yml` under a meaningful group with a proper tag name.

![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/61202986/e3e47353-1380-44af-8ca9-f4e8cccb031b)

**Comment by osujipius** [comment]
Can I pick this up? @ragesoss

**Comment by ragesoss** [comment]
@osujipius yes. I recommend choosing a very limited number of cases, so that your changes are limited to a few files.

**Comment by osujipius** [comment]
> @osujipius yes. I recommend choosing a very limited number of cases, so that your changes are limited to a few files.

Got it!
Thanks
